### PR TITLE
[AP-512] Fastsync mysql BINARY into snowflake BINARY column type

### PR DESCRIPTION
**Pull request by koszti**
## Description

mysql-to-snowflake fastsync should load `BINARY` mysql column types into snowflake `BINARY` column type. The latest 0.12.2 does not do this: https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/mysql_to_snowflake.py#L43

This is most probably a bug created accidentally by a big PR that fixed pylint and code formatting issues in the entire pipelinewise codebase:  https://github.com/transferwise/pipelinewise/commit/3aa15a03931d2badd8d444a3bab9e0898c27aec6#diff-cb69f4bfe2a71a1cc8b2301f40ac0fa0L44

Problem also reported in #321 

## Checklist

- [x] Description above provides context of the change
- [ ] Unit tests coverage for changes (not needed for documentation changes)
- [x] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [x] Commit message/PR title starts with `[AP-NNNN]` if applicable. AP-NNNN = JIRA ID
- [x] Branch name starts with `AP-NNN` if applicable. AP-NNN = JIRA ID
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: pipelinewise/fastsync/mysql_to_snowflake.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def tap_type_to_target_type(mysql_type, mysql_column_type):
         'char': 'VARCHAR',
         'varchar': 'VARCHAR',
         'binary': 'VARCHAR',
-        'varbinary': 'VARCHAR',
-        'blob': 'VARCHAR',
+        'varbinary': 'BINARY',
+        'blob': 'BINARY',
```
Comment (Samira-El) [inline pipelinewise/fastsync/mysql_to_snowflake.py]: This should be kept Varchar

File: pipelinewise/fastsync/mysql_to_snowflake.py [hunk 2]
```diff
@@ -41,8 +41,8 @@ def tap_type_to_target_type(mysql_type, mysql_column_type):
         'char': 'VARCHAR',
         'varchar': 'VARCHAR',
         'binary': 'VARCHAR',
```
Comment (Samira-El) [inline pipelinewise/fastsync/mysql_to_snowflake.py]: This needs to be BINARY

File: pipelinewise/fastsync/mysql_to_snowflake.py (hunk 1 above)
Comment (koszti) [reply pipelinewise/fastsync/mysql_to_snowflake.py]: thank you, fixed by 8a510713d6c4cf61bdcc1d039ca9dac35d40dc16

File: pipelinewise/fastsync/mysql_to_snowflake.py (hunk 2 above)
Comment (koszti) [reply pipelinewise/fastsync/mysql_to_snowflake.py]: thank you, fixed by 8a510713d6c4cf61bdcc1d039ca9dac35d40dc16

**Pull request merged**